### DISCUSSION TITLE
Enhancement: When uploading PDF files, Form Builder can recognize existing Fields in the file.

### DISCUSSION THREAD

**Discussion by GXY2017** [docusealco/docuseal#230] [Ideas]
For example, this w8-BEN pdf has pre-set the fillable form and fields.  I don't want to create all fields again in Form Builder.
[FI09 -W8-BEN.pdf](https://github.com/docusealco/docuseal/files/14370793/FI09.-W8-BEN.pdf)

**Comment by nixbitpie** [comment]
That would be nice.

**Comment by omohokcoj** [comment]
Moving this to discussions/ideas

**Comment by shoaibsharif** [comment]
I second to this. Can be done either from frontend or backend, but would be nice if we can achieve this via API. right now I have to convert a pdf to html add docuseal form element to make it fillable. When convert back to pdf, that doesn't always look same as original pdf.
